internal/github/checkPendingCI.go

```diff
@@ -81,10 +81,11 @@ func getLastCommitTime(client *github.Client, owner, repo string, prNumber int)
 
 	// Since GetDate() returns a Timestamp (not *Timestamp), use the address to call GetTime()
 	commitTime := commitDate.GetTime() // Correctly accessing GetTime(), which returns *time.Time
-
 	if commitTime == nil {
 		return time.Time{}, fmt.Errorf("commit time is nil for PR #%d", prNumber)
 	}
+	log.Printf("The last commit time is %s", commitTime.Format(time.RFC3339))
```

Comment:
Maybe add more identifying context to this, like PR number and/or repo name? Logs are all on their own lines and should have self contained context because if goroutines are added to this, the logs that happened just before this are meaningless.

Reply:
This comment probably goes for other loglines too.